test/connectors/WireMockSuite.scala

```diff
@@ -36,7 +36,8 @@ trait WiremockSuite extends BeforeAndAfterAll with BeforeAndAfterEach {
   protected lazy val app: Application =
     new GuiceApplicationBuilder()
       .configure(
-        portConfigKey -> server.port().toString
+          "metrics.jvm" -> false,
```

Comment:
Is this setting required? It's not set in arrivals backend

Reply:
There are deviations between the two, but this was a solution to a metrics injection issue for our tests. We use it in a number of places in this api for tests. Only for the tests though